File: benchmarks/bench_sum.py
import time

nums = list(range(1, 10_000_001))

start = time.perf_counter()
total = sum(nums)
elapsed = (time.perf_counter() - start) * 1000

print(f"{elapsed:.4f}")
print(f"sum = {total}", file=__import__('sys').stderr)

File: benchmarks/bench_sum.rs
use std::time::Instant;

fn main() {
    let nums: Vec<i32> = (1..=10_000_000).collect();

    let start = Instant::now();
    let total: i64 = nums.iter().map(|&n| n as i64).sum();
    let elapsed = start.elapsed();

    println!("{}", elapsed.as_secs_f64() * 1000.0);
    eprintln!("sum = {total}");
}
